deploy/scripts/src/ipv-core/test.ts

```diff
@@ -222,12 +294,13 @@ export function coreScenario1 (): void {
   sleep(Math.random() * 3)
 
   group(
-    'B01_Core_05_PassportDataContinue POST',
+    'B01_Core_06_PassportDataContinue POST',
     function () {
       const startTime1 = Date.now()
       res = http.post(
         passportStubURL,
         {
+          sort: 'Kenneth Decerqueira (Valid Experian) Passport',
```

Comment:
```suggestion
          sort: user.passport,
```
Suggestion: Paramaterise these values

Reply:
Done